model/character.go

```diff
@@ -106,43 +104,41 @@ func (player *CharacterStats) DeterminePhysicalDamage() {
 		}
 	}
 
-	player.PhysicalDamage = int(damage)
+	player.PhysicalDamage = damage
 }
 
-// ChangeXP changes XP player from XPAmount, could be negative, return true if leveled up
-func (player *CharacterStats) ChangeXP(XPAmount int) bool {
-	player.CurrentXP = player.CurrentXP + XPAmount
+// ChangeXP changes XP player from XPAmount, could be negative, return true if leveled up.
+func (player *CharacterStats) ChangeXP(xpAmount int) bool {
+	player.CurrentXP += xpAmount
 	// Since we change XP, check if level changes
 	return player.DetermineLevel()
 }
```

Comment:
```go 
// ChangeXP changes XP player from XPAmount, could be negative, return true if leveled up
func (player *CharacterStats) ChangeXP(XPAmount int) bool {
	player.CurrentXP = player.CurrentXP + XPAmount
	// Since we change XP, check if level changes
	return player.DetermineLevel()
}
```

> model/character.go:113:40: captLocal: `XPAmount' should not be capitalized (gocritic)
func (player *CharacterStats) ChangeXP(XPAmount int) bool {

> model/character.go:114:2: assignOp: replace `player.CurrentXP = player.CurrentXP + XPAmount` with `player.CurrentXP += XPAmount` (gocritic)
        player.CurrentXP = player.CurrentXP + XPAmount